<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Keeping</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3">
        <widget class="QComboBox" name="comboBoxExpenseCategory"/>
       </item>
       <item row="2" column="5">
        <widget class="QComboBox" name="comboBoxRevenueName"/>
       </item>
       <item row="1" column="0">
        <widget class="QDateEdit" name="dateEditFrom">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QComboBox" name="comboBoxAssetName"/>
       </item>
       <item row="1" column="6">
        <widget class="QComboBox" name="comboBoxAssetCategory"/>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Asset</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditDescriptionFilter">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Expense</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QComboBox" name="comboBoxRevenueCategory"/>
       </item>
       <item row="1" column="7">
        <widget class="QComboBox" name="comboBoxLiabilityCategory"/>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Liability</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Revenue</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QDateEdit" name="dateEditTo">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QComboBox" name="comboBoxLiabilityName"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Description Filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Time Range</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="comboBoxExpenseName"/>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="comboBoxExpenseHousehold"/>
       </item>
       <item row="3" column="5">
        <widget class="QComboBox" name="comboBoxRevenueHousehold"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonMerge">
      <property name="text">
       <string>Merge Transaction</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Delete Transactions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFunctions">
    <property name="title">
     <string>Functions</string>
    </property>
    <addaction name="actionAddTransaction"/>
    <addaction name="actionAccountManager"/>
    <addaction name="actionFinancialStatement"/>
    <addaction name="actionInvestmentAnalysis"/>
    <addaction name="actionTransactionValidation"/>
    <addaction name="actionHouseholdManager"/>
   </widget>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFunctions"/>
   <addaction name="menuLogin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddTransaction"/>
   <addaction name="actionAccountManager"/>
   <addaction name="actionFinancialStatement"/>
   <addaction name="actionInvestmentAnalysis"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAddTransaction">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/AddTransaction.png</normaloff>:/icons/AddTransaction.png</iconset>
   </property>
   <property name="text">
    <string>Add Transaction</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAccountManager">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/Transactions.png</normaloff>:/icons/Transactions.png</iconset>
   </property>
   <property name="text">
    <string>Account Manager</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionInvestmentAnalysis">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/Investment_Analysis.png</normaloff>:/icons/Investment_Analysis.png</iconset>
   </property>
   <property name="text">
    <string>Investment Analysis</string>
   </property>
   <property name="toolTip">
    <string>Investment Analysis</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionFinancialStatement">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/Balance_Sheet.png</normaloff>:/icons/Balance_Sheet.png</iconset>
   </property>
   <property name="text">
    <string>Financial Statement</string>
   </property>
   <property name="toolTip">
    <string>Financial Statement</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionTransactionValidation">
   <property name="text">
    <string>Transaction Validation</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionHouseholdManager">
   <property name="text">
    <string>Household Manager</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Log in</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
